schemas/session-replay/mobile/image-wireframe-schema.json

```diff
@@ -0,0 +1,33 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "session-replay/mobile/image-wireframe-schema.json",
+  "title": "ImageWireframe",
+  "type": "object",
+  "description": "Schema of all properties of a ImageWireframe.",
+  "allOf": [
+    {
+      "$ref": "_common-shape-wireframe-schema.json"
+    },
+    {
+      "required": ["type", "base64", "imageFileExtension"],
+      "properties": {
+        "type": {
+          "type": "string",
+          "description": "The type of the wireframe.",
+          "const": "image",
+          "readOnly": true
+        },
+        "base64": {
+          "type": ["string", "null"],
+          "description": "base64 representation of the image. Can have a null value",
+          "readOnly": false
+        },
+        "imageFileExtension": {
```

Comment:
suggestion(blocking): why not sending a `contentType` attribute instead? this is assuming the image has a filename, which may not.

I'd like to hear @mariusc83 thoughts on this.

Reply:
On iOS we will be always sending PNGs (at least in the first iteration).

Reply:
Okay, I'm thinking mime-types can be more cross-platform and future proof.

Reply:
👍 for `mimeType: String`, "extension" is vague

Reply:
**suggestion(not-blocking):** I would go the extra mile and type this field with an enum with all the possible types we can expect See list here: https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types

Reply:
+1 for `mimeType` and for @ThibautGeriz suggestion...even though there are lots of extensions there. Just FYI from Android I expect to send `.webp` ;)

Reply:
For the naming I opened a thread to discuss it. For the type guard of the possible value, I think it a good move but I would do it in a second step not to add more constraints on the POC implementation